src/components/Company/CompanyName/CorrectNameOptions.vue

```diff
@@ -67,23 +80,28 @@ export default class CorrectNameOptions extends Vue {
   // local properties
   private displayedOptions: Array<CorrectNameOptionIF> = []
   private panel = null as number
-  private submit = false
+  private formType = ''
+  private currentFormType = ''
```

Comment:
Are these strings or CorrectionTypes? If the latter then init as null?
See also `identifyForm()` below.

Reply:
Will update